.drone.yml

```diff
@@ -38,6 +38,22 @@ steps:
     - tag
 
 - name: docker-publish
+  image: plugins/docker
+  settings:
+    dockerfile: package/Dockerfile
+    password:
+      from_secret: docker_password
+    repo: "rancher/local-path-provisioner"
+    tag: "${DRONE_BRANCH}-head-amd64"
+    username:
+      from_secret: docker_username
+  when:
```

Comment:
Is this change applying to all branches or actual master specific? If it's for master only, probably need a specific condition.